addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14087,7 +14087,21 @@ msgctxt "#36043"
 msgid "No CEC capable TV detected. Disable polling for CEC capable devices?"
 msgstr ""
 
-#empty strings from id 36044 to 36100
+#. Desc of action to be taken when "changing CEC source" (2nd)
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36044"
+msgid "Stop Playback"
+msgstr ""
+
+#. Desc of action to be taken when "changing CEC source" (3rd)
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
```

Comment:
the "usage comment" is the "#: foo/bar.cpp" part. That's mainly for internal use.